DNX.sln

```diff
@@ -117,6 +117,10 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Dnx.DesignTimeHos
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Dnx.Runtime.Json.Sources", "src\Microsoft.Dnx.Runtime.Json.Sources\Microsoft.Dnx.Runtime.Json.Sources.xproj", "{F88EB033-3BCA-411A-987F-B705569F9317}"
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Dnx.Testing.CommonUtils", "test\Microsoft.Dnx.Testing.CommonUtils\Microsoft.Dnx.Testing.CommonUtils.xproj", "{00E320EE-F8E7-40C2-9334-A02561EDAEAA}"
+EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Dnx.Testing.SampleTests", "test\Microsoft.Dnx.Testing.SampleTests\Microsoft.Dnx.Testing.SampleTests.xproj", "{88FF8911-526C-4FBE-B4E2-707CDAF1DEC0}"
```

Comment:
I don't think these tests should be official for now. Instead these should serve as a template for porting over the existing set of tests. These should be removed once the old tests are ported over.